class_person.rb

```diff
@@ -1,14 +1,19 @@
-class Person
+require_relative 'class_nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
 
-  def initialize(age, name = 'Unknown', parent_permission: true)
-    @id = Random.rand(1..1000)
+  def initialize(age, name, parent_permission: true)
+    super()
```

Comment:
- [OPTIONAL] I think it would be better to use a default value for name as we were required to do in the previous milestone 💪 Since the previous reviewer didn't mentioned about the default value, therefore I'm making it as optional, however, I highly recommend fixing this ⛏🤓

Reply:
- Add default value for name parameter to resolve bug